lib/SPIRV/LLVMToSPIRVDbgTran.cpp

```diff
@@ -1387,6 +1390,49 @@ SPIRVExtInst *LLVMToSPIRVDbgTran::getSource(const T *DIEntry) {
   return Source;
 }
 
+template <class T>
```

Comment:
We probably know, that it's compilation unit, so might be no need for template

Reply:
Removed template